content/app/content_main_runner_impl.cc

```diff
@@ -998,6 +1024,35 @@ int ContentMainRunnerImpl::Initialize(ContentMainParams params) {
 
   delegate_->PreSandboxStartup();
```

Comment:
I think we should put the SbPlayer creation in  
this PreSandboxStartup() which is implemented  
in the embedder (i.e. //cobalt, see [1]). This  
is to avoid modifying the //content contents :)


[1] https://source.chromium.org/search?q=PreSandboxStartup%5C(&sq=&ss=chromium